src/greptimeai/collector/__init__.py

```diff
@@ -81,3 +81,15 @@ def collect_metrics(self, span_attrs: Dict[str, Any], attrs: Optional[Attributes
             completion_cost=completion_cost,
             attrs=attrs,
         )
+
+    def add_span_event(
```

Comment:
if this method has the same signature of `_collector.add_span_event`, we can just call it without a new method.

If a new method will make method signature cleaner, maybe we can give a new one